src/CloudModelGenerator/Program.cs

```diff
@@ -20,9 +28,19 @@ static int Main(string[] args)
             var includeTypeProvider = app.Option("-t|--withtypeprovider", "Indicates whether the CustomTypeProvider class should be generated.", CommandOptionType.NoValue);
             var structuredModel = app.Option("-s|--structuredmodel", "Indicates whether the classes should be generated with types that represent structured data model.", CommandOptionType.NoValue);
 
+
             app.OnExecute(() =>
             {
-                if (!projectIdOption.HasValue())
+                // Check if default values are set
+                var passedSetProjectId = configuration["defaultFlags:0:projectId"] ?? projectIdOption.Value();
```

Comment:
the command line params should always take precedence -> switch the right and left side of the `??` operator